.github/workflows/internal-images.yml

```diff
@@ -35,8 +35,9 @@ jobs:
           - dockerfile: |
               FROM centos:8
               RUN yum install -y --quiet make which git gcc && yum clean all
-              RUN curl -sL https://raw.githubusercontent.com/travis-ci/gimme/master/gimme | GIMME_GO_VERSION=1.17.x bash
```

Comment:
a push to master made hard-coded this, which made the workflow succeed. I tested the below change in a private repo after realizing interpolation was wonky and might take a commit or two to figure out.